Add advice on building all modules on Debian-like (#673)
python/devguide: setup.rst

If that package is not available for your system, try reducing the minor
version until you find a package that is available.

If you want to build all modules, install these packages in the versions you

> prefer, changing the version name as above::

Reviewer: I think this can be combined with the above point, as I'm pretty sure the explicit package names cover everything that would be installed by `build-dep`.  The above point should also be updated to a more recent version (or the minor version removed entirely, and the suggestion amended to suggest adding a minor version if `python3` doesn't cut it).

Author: I have removed the minor version from the Python package for which the command installs the build dependencies.

> If you want to build all modules, install these packages in the versions you
> prefer, changing the version name as above::

Reviewer: I'm also not a fan of the "versions you prefer..." bit, I would suggest to trim it back to just 
```suggestion
If you want to build all optional modules, install the following packages and
their dependencies::
```

Reviewer 2: +1

Author: I have accepted this suggestion entirely, merging it with the deletion of the previous paragraph.

   $ sudo apt-get install build-essential gdb lcov libbz2-dev libffi-dev \
         libgdbm-dev liblzma-dev libncurses5-dev libreadline6-dev \

>          libsqlite3-dev libssl-dev lzma lzma-dev tk-dev uuid-dev xvfb \

Reviewer: `xvfb` is not actually necessary, it would only allow you to run Tk tests on a headless machine.

Author: I have removed `xvfb` from the list.
